1001-alerts-design-guidance.md

```diff
@@ -0,0 +1,57 @@
+---
+category: component design guidance
+state: draft
+created: 2024-10-21
+updated: 2024-10-21
+---
+
+# Alert Design Guidance
+
+The [Alert](https://clarity.design/documentation/alert) component implements banners that demand the user's attention for important messages.
+
+## Guidance
+
+### App Level Alerts
+
+App Level Alerts support only `Error`, `Warning`, and `Info` statuses.
+
+- **Must** be used exclusively for the global context of an application.
+- **Must** be placed at the top of **all** content. 
+- **Must** NOT be used to convey the success of an operation.
+- Alerts for communicating operation success **must** appear in context or as a notification message.
+
+### Standard Alerts
+
+Standard Alerts support `Error`, `Warning`, `Info`, and `Success` statuses.
+
+- **Must** use `Compact` alerts only in places where vertical space is scarce, and information density is needed; otherwise, the `default` size **should** be used.
```

Comment:
```suggestion
- **May** use lightweight alerts in places where vertical space is scarce and information density is needed; otherwise, the default size **should** be used.
```

Change "must only" language to "may" and remove inline code.